### ISSUE TITLE
xp-pen g430s "Tablet not found"

### ISSUE THREAD

**Issue by Tomycj** [InfinityGhost/TabletDriver#15]
Hawku drivers don't seem to work on the xp-pen g430s. Tried both 0.3 and 0.2.3 and I get the same error.

Obviously I uninstalled all previous drivers and checked that nothing remained in the device manager before installing this one. Bought it a few weeks ago from here: https://www.aliexpress.com/item/32609175289.html

In the old hawku driver (0.2.3) github there are plenty of issues with this tablet, please help us :c
https://www.reddit.com/r/osugame/comments/co7p6z/need_help_fixing_my_xppen_g430/

## Hardware/software information
Windows Version: 10 (home)
Tablet Driver version: 0.3 (also happens in 0.3pre and 0.2.3)
Tablet model: g430s

## Logs
[console.txt](https://github.com/InfinityGhost/TabletDriver/files/3665086/console.txt)

Debug in 0.2.3 (in 0.3 debug option doesn't change anything in the console output):
[console debug.txt](https://github.com/InfinityGhost/TabletDriver/files/3665087/console.debug.txt)

**Comment by InfinityGhost** [comment]
I'm not 100% sure why the older G430s doesn't work but I would definitely like to look into it more. If you could join my [discord](https://discord.gg/P3GYdUS), when I have some free time I could look into it directly with you.

**Comment by InfinityGhost** [comment]
Will look over this issue again before I release v0.3.1

**Comment by xCuri0** [comment]
uninstall xp-pen drivers and add this to xp-pen.cfg
```
#
# XP-Pen G430S_B
#
HIDTablet 0x28BD 0x0913 0xFF0A 0x0001
CheckString Product "4 inch PenTablet"
Name "XP-Pen G430S_B"
ReportId 0x02
ReportLength 12
DetectMask 0x80
MaxX 10160
MaxY 7620
MaxPressure 8191
Width 101.6
Height 76.2
InitReport 0x02 0xB0 0x04 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00
```

**Comment by Tomycj** [comment]
Thank you! It worked. Turns out my tablet is a G430S_B, but it only figured in a sticker in the back. I don't know if the config is perfect  (I was expecting things like maxY to be the same as the G430_S), but now at least the tablet is detected and seems to work fine.

**Comment by InfinityGhost** [comment]
> Thank you! It worked. Turns out my tablet is a G430S_B, but it only figured in a sticker in the back. I don't know if the config is perfect (I was expecting things like maxY to be the same as the G430_S), but now at least the tablet is detected and seems to work fine.

So that's the actual name of the tablet? I wanted to make sure before I merged anything.

**Comment by Tomycj** [comment]
Everything said "XP-Pen G430S" Except for the little sticker. It reads "SKU: StarG430S_B". So it looks like new G430Ss (recently bought) are the B variant, yes.

**Comment by InfinityGhost** [comment]
I will be merging #25 soon, which should close this issue. This change be released with v0.3.1

**Comment by InfinityGhost** [comment]
Merged #25, closing this issue unless it arises again.
